partner_view_360/models/res_partner.py

```diff
@@ -337,12 +338,26 @@ def create(self, values):
 
     @api.model
     def search_pnr(self, pnr):
+        now_year = str(datetime.now().year)
+        before_year = str(datetime.now().year - 100)
```

Comment:
before_year is a pretty ambiguous variable name, consider changing to last_century or something more descriptive.